test/test_files/patcher_data/dread/starter_preset.json

```diff
@@ -3659,10 +3659,10 @@
         }
     ],
     "text_patches": {
+        "GUI_DIFSELECTOR_LABEL_DESCRIPTOR_EASY": "Words Hash ($$$$$)",
```

Comment:
I wonder why the JSON is not stable. I guarantee the next time I change something it pop ups in a third place.